calico/reference/resources/bgppeer.mdx

```diff
@@ -60,6 +60,12 @@ the [node resource](node.mdx).
 
 :::
 
+:::note
+
+If you want to use the non-default BGP Peer parameters, you must disable node-to-node mesh by setting the `nodeToNodeMeshEnabled` parameter to false in the [BGPConfiguration](./bgpconfig) resource.
```

Comment:
Can I suggest instead "If you want to use BGPPeer resources to configure the BGP sessions between nodes within the cluster - for example, in order to set non-default BGPPeer parameters - you should disable the node-to-node mesh ..."